config/routes.rb

```diff
@@ -8,6 +8,10 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] Please kindly include a `new` route to be handled by the `new` method in the `CommentsController`